Misc/NEWS.d/next/Library/2019-01-18-23-10-10.bpo-23078.l4dFoj.rst

```diff
@@ -0,0 +1,2 @@
+Add support for staticmethod and classmethod with autospec. Based on patch
+by Felipe.
```

Comment:
You might want to include Felipe's surname. And you can also get credit for this :).

Reply:
Thanks, made the changes. cc : @felipeochoa